pyproject.toml

```diff
@@ -38,6 +38,25 @@ classifiers = [
     "Topic :: Utilities",
 ]
 
+dependencies = [
+    "importlib-resources >= 6.0; python_version < '3.10'",
+    "setuptools",
```

Comment:
Why is `setuptools` added here? It's only a build-system requirement and not used during the runtime AFAICT.

Reply:
Agree, this is a refuse, I can remove it. It was used in past cases